queens/iterators/control_variates_iterator.py

```diff
@@ -0,0 +1,248 @@
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (c) 2025, QUEENS contributors.
+#
+# This file is part of QUEENS.
+#
+# QUEENS is free software: you can redistribute it and/or modify it under the terms of the GNU
+# Lesser General Public License as published by the Free Software Foundation, either version 3 of
+# the License, or (at your option) any later version. QUEENS is distributed in the hope that it will
+# be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details. You
+# should have received a copy of the GNU Lesser General Public License along with QUEENS. If not,
+# see <https://www.gnu.org/licenses/>.
+#
+"""Monte Carlo Control Variates Iterator."""
+
+import logging
+
+import numpy as np
+
+from queens.iterators.iterator import Iterator
+from queens.utils.logger_settings import log_init_args
+from queens.utils.process_outputs import write_results
+
+_logger = logging.getLogger(__name__)
+
+
+class ControlVariatesIterator(Iterator):
+    r"""Monte Carlo control variates iterator.
+
+    The control variates method in the context of Monte Carlo is used to quantify uncertainty in a
+    model when input parameters are uncertain and only expressed as probability distributions. The
+    Monte Carlo control variates method uses so-called low-fidelity models as control variates to
+    make the quantification more precise. In the context of Monte Carlo, the control variate method
+    is sometimes also called control variable method.
+
+    The estimator for the Monte Carlo control variates method with a single control variate is
+    given by
+    :math:`\hat{\mu}_{f}= \underbrace{\frac{1}{N} \sum\limits_{i=1}^{l} \Big [ f(x^{(i)}) - \alpha
+    \Big(g(x^{(i)}) - \hat\mu_{g} \Big) \Big]}_\textrm{cross-model estimator}`
+    where :math:`f` represents the model and :math:`g` the control variate. :math:`N` represents
+    the number of samples on the cross-model estimator and :math:`x^{(i)}` are random
+    parameter samples.
+
+    In case the mean of the control variate is known, :math:`\hat\mu_{g}` can be passed to the
+    iterator as ´´expectation_cv´´. Otherwise, :math:`\hat\mu_{g}` is estimated with the Monte
```

Comment:
```suggestion
    iterator as expectation_cv. Otherwise, :math:`\hat\mu_{g}` is estimated with the Monte
```
I thought this would work but it doesn't, sorry about that.